integration/tests/crud/creategetdelete_test.go

```diff
@@ -155,6 +157,20 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
+		Context("and describe the stack for the cluster", func() {
+			It("should describe the cluster's stack", func() {
+				cmd := params.EksctlUtilsCmd.WithArgs("describe-stacks", "--cluster", params.ClusterName, "-o", "yaml")
+				session := cmd.Run()
+				Expect(session.ExitCode()).To(BeZero())
+				var stacks []*cloudformation.Stack
+				Expect(yaml.Unmarshal(session.Out.Contents(), &stacks)).To(Succeed())
+				Expect(len(stacks) > 1).To(BeTrue())
+				for _, s := range stacks {
+					Expect(aws.StringValue(s.StackName)).To(ContainSubstring(params.ClusterName))
+				}
+			})
+		})
```

Comment:
does `describe-stacks` return all stacks, e.g. stack for the cluster, stack for the nodegrou, stack for an IRSA etc? If it does might be worth expanding the test a bit to check that

Reply:
It returns all stacks.

Reply:
can we update this test to check that logic as well then?

Reply:
`Expect(len(stacks) > 1).To(BeTrue())` - fails if there is only one :)

Reply:
> 

that isn't a very explicit test though...

Reply:
But I don't wannnnaaa

![tony](https://user-images.githubusercontent.com/182850/143842351-e4c93c29-3e14-463a-a1ab-8acf5c1896af.jpeg)

Reply:
Fiiiiineee. I'll do it. I'll update it. Uuuuhh. Making me work on a Monday... *grumble*, *grumble*

Reply:
I agree that we should expand this test to ensure stacks for all resources are returned.

Reply:
On it, on it. :D I'll do this right after I have the config-file thing working right. :D